diff --git a/ext/meson/mesonbuild/modules/gnome.py b/ext/meson/mesonbuild/modules/gnome.py
index db85420..c89e575 100644
--- a/ext/meson/mesonbuild/modules/gnome.py
+++ b/ext/meson/mesonbuild/modules/gnome.py
@@ -191,7 +191,10 @@ class GnomeModule(ExtensionModule):
         else:
             depfile = kwargs['output'] + '.d'
             kwargs['depfile'] = depfile
-            kwargs['command'] = copy.copy(cmd) + ['--dependency-file', '@DEPFILE@']
+
+            # sqlnotebook change, remove glib-compile-resources flag "--dependency-file" because it is not understood
+            # by the version of that tool included in ubuntu 16.04
+            kwargs['command'] = cmd
         target_c = GResourceTarget(name, state.subdir, state.subproject, kwargs)
 
         if gresource: # Only one target for .gresource files
