diff --git a/ext/meson/mesonbuild/compilers/c.py b/ext/meson/mesonbuild/compilers/c.py
index 4c6e3a2..fd82e51 100644
--- a/ext/meson/mesonbuild/compilers/c.py
+++ b/ext/meson/mesonbuild/compilers/c.py
@@ -236,6 +236,9 @@ class CCompiler(Compiler):
         if pc.returncode != 0:
             raise EnvironmentException('Compiler {0} can not compile programs.'.format(self.name_string()))
         # Run sanity check
+
+        return # sqlnotebook addition, skip the sanity check
+
         if self.is_cross:
             if self.exe_wrapper is None:
                 # Can't check if the binaries run so we have to assume they do
