diff --git a/ext/railroad-diagrams/railroad_diagrams.py b/ext/railroad-diagrams/railroad_diagrams.py
index 27bbd31..721e354 100644
--- a/ext/railroad-diagrams/railroad_diagrams.py
+++ b/ext/railroad-diagrams/railroad_diagrams.py
@@ -5,10 +5,10 @@ if sys.version_info >= (3, ):
     unicode = str
 
 # Display constants
-VERTICAL_SEPARATION = 8
+VERTICAL_SEPARATION = 16
 ARC_RADIUS = 10
 DIAGRAM_CLASS = 'railroad-diagram'
-TRANSLATE_HALF_PIXEL = True
+TRANSLATE_HALF_PIXEL = False
 INTERNAL_ALIGNMENT = 'center'
 DEBUG=False
 
@@ -170,9 +170,7 @@ class Style(DiagramItem):
         return self
 
     def writeSvg(self, write):
-        # Write included stylesheet as CDATA. See https://developer.mozilla.org/en-US/docs/Web/SVG/Element/style
-        cdata = u'/* <![CDATA[ */\n{css}\n/* ]]> */\n'.format(css=self.css)
-        write(u'<style>{cdata}</style>'.format(cdata=cdata))
+        pass
 
 
 class Diagram(DiagramItem):
