diff --git a/ext/uncrustify/src/space.cpp b/ext/uncrustify/src/space.cpp
index 53f8d5a..83de91d 100644
--- a/ext/uncrustify/src/space.cpp
+++ b/ext/uncrustify/src/space.cpp
@@ -148,6 +148,19 @@ static argval_t do_space(chunk_t *first, chunk_t *second, int &min_sp, bool comp
    LOG_FMT(LSPACE, "%s(%d): orig_line is %zu, orig_col is %zu, first->text() '%s', type is %s\n",
            __func__, __LINE__, first->orig_line, first->orig_col, first->text(), get_token_name(first->type));
 
+   if (first->type == CT_QUESTION || second->type == CT_QUESTION)
+   {
+      return(AV_IGNORE);
+   }
+   if (first->type == CT_COLON || second->type == CT_COLON)
+   {
+      return(AV_IGNORE);
+   }
+   if (first->type == CT_ANGLE_CLOSE && second->type == CT_BRACE_OPEN)
+   {
+      return(AV_FORCE);
+   }
+
    min_sp = 1;
    if (first->type == CT_IGNORED || second->type == CT_IGNORED)
    {
